src/StyledLargeButton.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components/macro'
+
+const StyledLargeButton = styled.button`
```

Comment:
Here you use almost the exact styling as in your `StyledNextButton` in your `QuestionPage.js`. You could use props to alternate the styled accordingly. Doing so, you save yourself quite some repetition within your code & styles.

Reply:
Thank you for your suggestion Lene :-) I added a new Button Component and wrapped it in the styled-component's `styled()`constructor - I could get rid of a lot of code :D